Lib/inspect.py

```diff
@@ -1078,7 +1078,8 @@ def get_lineno(self):
 
             # First, let's see if there are any method definitions
             for member in self.cls.__dict__.values():
-                if isinstance(member, types.FunctionType):
+                if isinstance(member, types.FunctionType) and \
```

Comment:
Please use parentheses instead of a backslash line continuation here, per the recommendations of PEP 8: https://peps.python.org/pep-0008/#maximum-line-length

Reply:
Fixed